package.json

```diff
@@ -44,6 +44,6 @@
 		"access": "public"
 	},
 	"volta": {
-		"node": "14.21.3"
```

Comment:
Was accidentally downgraded in https://github.com/eps1lon/types-react-codemod/pull/209